sources/packages/backend/libs/utilities/src/esdc-utils.ts

```diff
@@ -43,11 +47,11 @@ export function getPartTimeMaritalStatusCode(
 
 /**
  * Get the e-Cert flag for the borrower's persistent or prolonged disability status.
- * @param hasPDorPPD indicates if a borrower has a persistent or prolonged disability.
+ * @param hasPPD indicates if a borrower has a persistent or prolonged disability.
  * @returns "Y" or "N" flag.
  */
-export function getPPDFlag(hasPPD?: boolean): string {
-  return hasPPD === true ? "Y" : "N";
+export function getPPDFlag(hasPPD?: ApplicationDisabilityStatus): string {
```

Comment:
Is still the argument optional?

Also hasPPD or even the function name can be renamed right?

Reply:
Seems to me that the function name is right since it is not returning a Boolean.

Reply:
I changed the argument name to `applicationDisabilityStatus`.